### PR TITLE
Update symfony_requirements

### PR DESCRIPTION
**Pull request by JarJak**
fix symfony version

### CODE REVIEW

**Comment by hacfi** [comment]
@JarJak Good catch! I think they prefer just `Symfony` though. Let’s wait for feedback though.

File: bin/symfony_requirements [hunk 1]
```diff
@@ -43,9 +43,9 @@ foreach ($symfonyRequirements->getRecommendations() as $req) {
 }
 
 if ($checkPassed) {
-    echo_block('success', 'OK', 'Your system is ready to run Symfony2 projects');
```
Comment (hacfi) [inline bin/symfony_requirements]: While you’re at it: you could update `/** @var $req Requirement */` to the more common variant `/** @var Requirement $req */` if you don’t mind

Comment (JarJak) [inline bin/symfony_requirements]: both are acceptable by IDE, i don't think your version is more common

**Comment by Tobion** [comment]
Already fixed in https://github.com/sensiolabs/SensioDistributionBundle/pull/247

File: bin/symfony_requirements (hunk 1 above)
Comment (hacfi) [inline bin/symfony_requirements]: @JarJak According to the specs at http://phpdoc.org/docs/latest/references/phpdoc/tags/var.html that is the recommended version and is consistent with `@param ClassName $var`.

**Comment by JarJak** [comment]
but it's not fixed in this repo

**Comment by Tobion** [comment]
it will be updated on the next vendor update automatically.

**Pull request closed without merging**